### PR TITLE
added justin as author, changed syntax to use lists, added datatype d…

### PR DESCRIPTION
**Pull request by jricher**
…efinitions

### CODE REVIEW

**Comment by tlodderstedt** [comment]
Looks pretty good. I have added some comments. The biggest open question is related to locations vs resource. 
I updated the acknowledgment section to reflect the new authorship.

File: mainmatter.md [hunk 1]
```diff
@@ -150,40 +151,12 @@ Here is an example:
             "access":[  
                "read"
             ]
```
Comment (tlodderstedt) [inline mainmatter.md]: looks incomplete

File: mainmatter.md [hunk 2]
```diff
@@ -309,19 +291,21 @@ This depends, however, on the AS to know what authorization details are relevant
 As an example, it is possible to specify that the client will get "read" access to "file X" stored at the resource "https://store.example.com". To achieve this, the example given above for access to an IMAP server is slightly modified to use the `resource` element as part of the top level claims within the `authorization_details` element. 
 
 ```JSON
-{  
-   "https://scheme.example.org/imap":{  
+[  
+   {
+      "type": "https://scheme.example.org/imap":
       "resource":"imap.example.com",
```
Comment (tlodderstedt) [inline mainmatter.md]: I think resource and locations are mutual exclusive, we need to decide what element to use and how - see also #7

File: mainmatter.md [hunk 3]
```diff
@@ -276,24 +254,28 @@ This is shown in the following example:
...
-           "instructedAmount":{  
-             "currency":"EUR",
-             "amount":"123.50"
-           },
-           "debtorAccount":{  
-             "iban":"DE40100100103307118608"
-           },
-           "creditorName":"Merchant123",
-             "creditorAccount":{  
+       "authorization_details":[
```
Comment (tlodderstedt) [inline mainmatter.md]: Can you please also update the URI query parameter example?

File: mainmatter.md (hunk 2 above)
Comment (jricher) [reply mainmatter.md]: I struggled with that, too. The thing is, resources can be logical names, and I kinda wanted location to be strictly URI's. So my thought was to leave them both in and let them be used side by side, for the moment, and then figure out what exactly the relationship and overlap is. 

You could make similar arguments about the PoP `aud` parameter, and likely a dozen proprietary extensions that do similar things. I don't think we can necessarily fold all of them in cleanly.

File: mainmatter.md (hunk 3 above)
Comment (jricher) [reply mainmatter.md]: I don't see the example you're referring to, I thought I'd updated all of them.

File: mainmatter.md (hunk 1 above)
Comment (jricher) [reply mainmatter.md]: This section is missing a `type` field, but it's an example of how to put a new data type field in place so I didn't think that was important. I've added the type field to the example.

Comment (jricher) [reply mainmatter.md]: Oh, I think I see what got removed now, added back.

**Pull request merged**